CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.17.4
+
+* make `Configuration::location()` public
+* export `ScopedHashMap` from the library
+* make `RequirePathLocator` and `PathLocator` public
+* make `PathRequireMode::initialize()` public
```

Comment:
Use a single entry like this. We can add the other thing we're making public later once we decide what we go with (see the related issue)
```suggestion
* mark some internal functionality public for the darklua library (`Configuration::location()`) ([#328](https://github.com/seaofvoices/darklua/pull/328))
```